README_libf3d.md

```diff
@@ -0,0 +1,209 @@
+# libf3d - A library to render 3D meshes
+
+By Michael Migliore and Mathieu Westphal.
+
+libf3d is a BSD-licensed C++ library to open and render 3D meshes. It is of course used by F3D.
+libf3d API is simple and easy to learn. Python bindings are provided through pybind11.
+
+# Getting Started
+
+Rendering a file and starting the interaction is very easy:
+
+```
+// Create a f3d::engine
+f3d::engine eng();
+
+// Recover the file loader
+f3d::loader& load = eng.getLoader();
+
+// Add a file to the loader
+load.addFile("path/to/file.ext");
+
+// Load the file
+load.loadFile();
```

Comment:
It would be great to call the API like this, but it will require changes in the loader API.

```suggestion
eng.getLoader()                 // Recover the file loader
   .addFile("path/to/file.ext") // Add a file to the loader
   .loadFile();                 // Load the file
```

Reply:
done in #342